_lua5.1-tests/calls.lua

```diff
@@ -215,6 +215,14 @@ assert(a==1 and b==1 and c==nil and d==nil)
 a = ret2{ unlpack{1,2,3}, unlpack{3,2,1}, unlpack{"a", "b"}}
 assert(a[1] == 1 and a[2] == 3 and a[3] == "a" and a[4] == "b")
 
+-- Any expression enclosed in parentheses always results in only one value.
+local function enclosed_ret() return (ret2(1, 2)) end
+a,b = enclosed_ret()
+assert(a == 1 and b == nil)
+
+local function enclosed_vararg(...) return (...) end
+a,b,c=enclosed_vararg(1, 2, 3)
+assert(a == 1 and b == nil and c == nil)
```

Comment:
This file is a copy of c-lua source code. Could you add these cases to `_glua-tests/issues.lua`?

Reply:
Done

Reply:
Added one more fix, rebased in single commit.